### PR TITLE
[Edge] add mqtt functions

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add internal functions to handle mqtt message.
This is initial commit for mqtt-hybrid feature after separating the edge library.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+/**
+ * @brief Subscribe a topic.
+ */
+int nns_edge_mqtt_subscribe (nns_edge_h edge_h);
+#else
+#define nns_edge_mqtt_connect(...) (NNS_EDGE_ERROR_NOT_SUPPORTED)
+#define nns_edge_mqtt_close(...) (NNS_EDGE_ERROR_NOT_SUPPORTED)
+#define nns_edge_mqtt_publish(...) (NNS_EDGE_ERROR_NOT_SUPPORTED)
+#define nns_edge_mqtt_subscribe(...) (NNS_EDGE_ERROR_NOT_SUPPORTED)
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_query/nnstreamer_edge_internal.h:64]: Suggestion:
```
static int nns_edge_mqtt_publish (nns_edge_h edge_h, const void *data, const int length)
{
  return NOT_SUPPORTED;
}
```
(Linux style)

File: gst/nnstreamer/tensor_query/nnstreamer_edge.h [hunk 2]
```diff
@@ -49,6 +49,7 @@ typedef enum {
   NNS_EDGE_ERROR_IO = -EIO,
   NNS_EDGE_ERROR_CONNECTION_FAILURE = -ECONNREFUSED,
   NNS_EDGE_ERROR_UNKNOWN = -(INT_MIN / 2),
+  NNS_EDGE_ERROR_NOT_SUPPORTED = -(NNS_EDGE_ERROR_UNKNOWN + 1),
```
Comment (myungjoo) [inline gst/nnstreamer/tensor_query/nnstreamer_edge.h:52]: ```s/+ 1/+ 2/``` for the compatibility with Tizen APIs?

**Review by myungjoo — COMMENTED** [review]
LGTM except for minor style suggestions.

File: gst/nnstreamer/tensor_query/nnstreamer_edge_internal.h (hunk 1 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/nnstreamer_edge_internal.h:74]: marked todo item.

File: gst/nnstreamer/tensor_query/nnstreamer_edge.h (hunk 2 above)
Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_query/nnstreamer_edge.h]: Updated

**Review by myungjoo — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**